### PR TITLE
Update esxi.py

### PR DESCRIPTION
**Pull request by ricobach**
Add build version of the host, so it would be possible to verify if there are updates available to the host

### CODE REVIEW

File: custom_components/esxi_stats/esxi.py [hunk 1]
```diff
@@ -132,6 +134,7 @@ def get_host_info(host):
         "name": host_name,
         "state": host_state,
         "version": host_version,
+        "version": host_build,
```
Comment (wxt9861) [inline custom_components/esxi_stats/esxi.py:137]: Please change the key version to say build so that we don't have a duplicate key.

**Review by wxt9861 — CHANGES_REQUESTED** [review]
Change version to build on line 137 to address a duplicate key.

File: custom_components/esxi_stats/esxi.py (hunk 1 above)
Comment (ricobach) [reply custom_components/esxi_stats/esxi.py]: Done ;)

**Review by wxt9861 — APPROVED** [review]

**Comment by wxt9861** [comment]
Thank you for your contribution!

**Pull request merged**